pyccel/codegen/printing/ccode.py

```diff
@@ -63,7 +63,9 @@
                   ('complex',4) : 'float complex',
                   ('int',4)     : 'int',
                   ('int',8)     : 'long',
-                  ('bool',1)    : 'int'}
+                  ('int',2)     : 'int',
+                  ('int',1)     : 'int',  
```

Comment:
Can you explain under what circumstances we deal with integers of 1 or 2 bytes? And why do we fall back to the standard 32-bit `int` in C in this case? Do you have any unit tests for this?